test/helpers.lua

```diff
@@ -89,7 +89,11 @@ function module.pcall_err(fn, ...)
     error('expected failure, but got success')
   end
   local errmsg = tostring(rv):gsub('^[^:]+:%d+: ', '')
-  return errmsg
+  if module.iswin() then
+    return errmsg:gsub('[a-zA-Z]:[^%[%]:]+:%d*:', '')
+  else
+    return tostring(errmsg):gsub('[^%[%]:]+:%d*:', '')
```

Comment:
I do not understand why this was changed at all. It's unrelated to the goal of this PR

Reply:
OK, I use 'pcall_err' with 'matches'.